pytao/interface_commands.py

```diff
@@ -2380,6 +2380,63 @@ def tao_global(tao, *, verbose=False, as_dict=True, raises=True):
     return __execute(tao, cmd, as_dict, raises, method_name='tao_global', cmd_type='string_list')
 
 
+def global_optimation(tao, *, verbose=False, as_dict=True, raises=True):
+    """
+    
+    Output optimization parameters.
+    Also see global:opti_de.
+    
+    Returns
+    -------
+    string_list
+    
+    Notes
+    -----
+    Command syntax:
+      python global:opti_de
+    
+    Output syntax is parameter list form. See documentation at the beginning of this file.
+    
+    Examples
+    --------
+    Example: 1
+     init: -init $ACC_ROOT_DIR/regression_tests/python_test/cesr/tao.init
+     args:
+    
+    """
+    cmd = f'python global:opti_de'
+    if verbose: print(cmd)
+    return __execute(tao, cmd, as_dict, raises, method_name='global_optimation', cmd_type='string_list')
+
+
+def global_opti_de(tao, *, verbose=False, as_dict=True, raises=True):
```

Comment:
This seems to be the same as above

Reply:
Hmm, that's giving the same as above.

Reply:
Okay, I found it. There were typos in `tao/tao_python_cmd.f90`.